app/services/pipeline_report_service.rb

```diff
@@ -67,13 +67,39 @@ def call
   end
 
   def generate
-    pipeline_run = PipelineRun.find(@pipeline_run_id)
     if @pipeline_run_id.nil?
-      Rails.logger.error("Pipeline run doesn't exist.")
-      return ""
-    elsif pipeline_run.total_reads.nil? || pipeline_run.adjusted_remaining_reads.nil?
-      Rails.logger.error("Pipeline run #{@pipeline_run_id} has no reads.")
-      return ""
+      return JSON.dump(
+        pipelineRunInfo: {
+          pipelineRunComplete: false,
+          pipelineRunFailed: false,
+          sampleStatus: "Waiting to Start or Receive Files",
+          errorMessage: nil,
+          knownUserError: nil,
+        },
+        metadata: {},
```

Comment:
Maybe use the metadata to show the status?

Reply:
The status is actually corresponding to the job status of the pipeline run, not the sample itself, so I felt it would fit in better with `pipelineRunInfo`. Renamed to `jobStatus` for more clarity.

Reply:
All this metadata is specific for the pipeline run (not sample). Don't forget  this is a pipeline run report service.

Reply:
Removed pipelineRunInfo and moved everything into metadata as discussed!